build/verify-archive.sh

```diff
@@ -2,8 +2,9 @@
 
 # This script sanity-checks a source tarball, assuming a Debian-based Linux
 # environment with a Go version capable of building CockroachDB. Source tarballs
-# are expected to build and install a functional cockroach binary into the PATH,
-# even when the tarball is extracted outside of GOPATH.
+# are expected to build, build after cleaning, and install a functional
```

Comment:
Is the purpose of the clean-then-build to ensure we don't clean up the new precompiled dependencies  (`embedded.go`, etc.)?

Is there any reason for doing the first build?

Reply:
Heh, good point. Changed to clean then build.